server/validators/helpers.ts

```diff
@@ -93,15 +95,19 @@ export const errorMessage = ({ path }) => {
 
     // Participant info
     eligibility:
-      "We're sorry, but current eligibility to work in Canada is a requirement to submit this form.",
-    preferredLocation: "Please select at least one location you'd like to work in.",
-    consent: "We're sorry, but we cannot process your request without permission.",
+      "We're sorry, but current eligibility to work in Canada is a requirement to submit this form",
+    educationalRequirements: `We're sorry, but this question is a requirement to submit this form`,
+    preferredLocation: "Please select at least one location you'd like to work in",
+    driverLicense: `We're sorry, but this question is a requirement to submit this form`,
```

Comment:
I'd remove We're sorry out of this one, unless you got that from the PEOI? Same with the education requirements one